src/pages/Proposal.vue

```diff
@@ -136,12 +136,13 @@ export default defineComponent({
 	setup() {
 		const formState = ProposalStore();
 
-		const onFinish = (values: any) => {
+		const onFinish = (values: Event) => {
```

Comment:
:+1: , we should really avoid using `any`

Reply:
Agreed